### PR TITLE
Automatically assign SGs to VMs in AWS

### PR DESCRIPTION
**Pull request by TuxPowered42**

### CODE REVIEW

**Comment by TuxPowered42** [comment]
retest this please

File: igvm/vm.py [hunk 1]
```diff
@@ -558,25 +622,25 @@ def aws_build(self,
...
         root_device = list(
-            ec2_res.images.filter(
+            self.ec2r.images.filter(
                 ImageIds=[self.dataset_obj['aws_image_id']]
             )
         )[0].root_device_name
         disk_size_gib = self.dataset_obj['disk_size_gib']
 
+        for aws_sg in self.aws_sgs:
+            self.dataset_obj['aws_security_group_ids'].add(aws_sg.group_id)
```
Comment (TuxPowered42) [inline igvm/vm.py:639]: @lacunoc Should we rather replace manually assigned groups instead of appending to them?

Comment (lacunoc) [reply igvm/vm.py:639]: Yes, we should.

File: tests/test_integration.py [hunk 2]
```diff
@@ -132,6 +132,7 @@ def setUp(self):
             self.vm_obj['aws_key_name'] = 'eu-central-1-key'
             self.vm_obj['service_groups'] = [
                 'igvm-integration.test.sg',
+                'ig-aws-base-internet.sg',
```
Comment (lacunoc) [inline tests/test_integration.py:135]: We maybe should add a TODO here to get rid of this group.

**Review by lacunoc — CHANGES_REQUESTED** [review]
beside of the minor comments, lgtm

File: igvm/vm.py (hunk 1 above)
Comment (TuxPowered42) [reply igvm/vm.py]: The attribute is now still populated and commited but overwritten. Removal of attributes will be in another MR.

File: tests/test_integration.py (hunk 2 above)
Comment (TuxPowered42) [reply tests/test_integration.py]: Done.

**Review by lacunoc — APPROVED** [review]

**Pull request merged**